Modules/MFT/src/BasicTrackQcTask.cxx

```diff
@@ -45,30 +44,46 @@ void BasicTrackQcTask::initialize(o2::framework::InitContext& /*ctx*/)
     ILOG(Info, Devel) << "Custom parameter - myOwnKey: " << param->second << ENDM;
   }
 
-  mMFT_xy_H = std::make_unique<TH2F>("mMFT_xy_H", "mMFT_xy_H", 22, -10.5, 10.5, 22, -10.5, 10.5);
-  getObjectsManager()->startPublishing(mMFT_xy_H.get());
-  getObjectsManager()->addMetadata(mMFT_xy_H->GetName(), "custom", "34");
+  mMFT_number_of_clusters_per_track_H = std::make_unique<TH1F>("mMFT_number_of_clusters_per_track_H", "mMFT_number_of_clusters_per_track_H", 16, -0.5, 15.5);
+  getObjectsManager()->startPublishing(mMFT_number_of_clusters_per_track_H.get());
+  getObjectsManager()->addMetadata(mMFT_number_of_clusters_per_track_H->GetName(), "custom", "34");
 
-  mMFT_pos_phi_H = std::make_unique<TH1F>("mMFT_pos_phi_H", "mMFT_pos_phi_H", 100, -3.145, 3.145);
+  mMFT_track_chi2_H = std::make_unique<TH1F>("mMFT_track_chi2_H", "mMFT_track_chi2_H", 81, -0.5, 80.5);
+  getObjectsManager()->startPublishing(mMFT_track_chi2_H.get());
+  getObjectsManager()->addMetadata(mMFT_track_chi2_H->GetName(), "custom", "34");
+
+  mMFT_charge_H = std::make_unique<TH1F>("mMFT_charge_H", "mMFT_charge_H", 3, -1.5, 1.5);
+  getObjectsManager()->startPublishing(mMFT_charge_H.get());
+  getObjectsManager()->addMetadata(mMFT_charge_H->GetName(), "custom", "34");
+
+  mMFT_pos_phi_H = std::make_unique<TH1F>("mMFT_pos_phi_H", "mMFT_pos_phi_H", 100, -3.2, 3.2);
   getObjectsManager()->startPublishing(mMFT_pos_phi_H.get());
   getObjectsManager()->addMetadata(mMFT_pos_phi_H->GetName(), "custom", "34");
 
-  mMFT_neg_phi_H = std::make_unique<TH1F>("mMFT_neg_phi_H", "mMFT_neg_phi_H", 100, -3.145, 3.145);
+  mMFT_neg_phi_H = std::make_unique<TH1F>("mMFT_neg_phi_H", "mMFT_neg_phi_H", 100, -3.2, 3.2);
   getObjectsManager()->startPublishing(mMFT_neg_phi_H.get());
   getObjectsManager()->addMetadata(mMFT_neg_phi_H->GetName(), "custom", "34");
```

Comment:
FYI, you don't need to add fake metadata to each object, unless this is what you want.

Reply:
I see, thanks for letting me know.